pkg/kubelet/kubelet.go

```diff
@@ -786,11 +787,26 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 
 	tokenManager := token.NewManager(kubeDeps.KubeClient)
 
+	var clusterTrustBundleManager clustertrustbundle.Manager
+	if kubeDeps.KubeClient != nil && utilfeature.DefaultFeatureGate.Enabled(features.ClusterTrustBundleProjection) {
+		kubeInformers := informers.NewSharedInformerFactoryWithOptions(kubeDeps.KubeClient, 0)
+		clusterTrustBundleManager, err = clustertrustbundle.NewInformerManager(kubeInformers.Certificates().V1alpha1().ClusterTrustBundles(), 256, 5*time.Minute)
```

Comment:
Is the cache size and TTL "good" ?

Reply:
I think the 5-minute TTL is good for now, since it's the mechanism by which space is reclaimed after a pod that used a CTB projection goes away.  I could imagine reorganizing the code so that we explicitly forget about pods as their volume is unmounted, and then the cache TTL could be infinite.

A similar consideration applies to the cache size --- really, we shouldn't have a max size and we should instead have as many entries as we need for the number of active CTB projections on the node.  For now, I have updated it to 2*maxPods, which is more flexible than a fixed number.